day-4/contract-tracker/src/App.js

```diff
@@ -0,0 +1,146 @@
+import React, { Component } from 'react';
+import logo from './logo.svg';
+import PropTypes from 'prop-types';
+import './App.css';
+
+class PlayerInput extends Component {
+  constructor (props) {
+    super(props);
+
+    this.state = {
+      title: "",
+      description: "",
+      price: "",
+      contract: {}
+    };
+
+    this.handleTitle = this.handleTitle.bind(this);
+    this.handleDescription = this.handleDescription.bind(this);
+    this.handlePrice = this.handlePrice.bind(this);
+    this.handleSubmit = this.handleSubmit.bind(this);
+  }
+
+  handleSubmit(event) {
```

Comment:
`handle[action]` is not super clear as to what it does. "Handling" is a very generic action. "Submit" is much clearer